samples/msgext-unfurling-ac-loop-components/nodejs/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "msgext-unfurling-ac-loop-components",
+  "version": "1.0.0",
+  "msteams": {
+    "teamsAppId": null
+  },
+  "description": "Messaging Extension Unfurling Adaptive Card Based Loop Components",
```

Comment:
Messaging Extension For Adaptive Card Based Loop Components

Reply:
Fixed